### PR TITLE
Add documentation for the release process

### PR DESCRIPTION
**Pull request by psavery**

### CODE REVIEW

File: docs/release/notarization/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Notarization on macOS requires several steps, and can be somewhat tedious to
+work through. The benefit, however, is that it guarantees a high level of
+security for the software.
+
+See [here](https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution) for more information.
+
+Steps
+=====
+
+1. Ensure you have a computer set up for notarization (FIXME: what do we do?)
```
Comment (psavery) [inline docs/release/notarization/README.md:13]: @cjh1 Any details we should add here?

File: docs/release/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Do some testing on each of these to verify that they work properly
+
+
+Windows Signing
+===============
+
+You will need to use [Microsoft's code signing](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/authenticode) with the `.msi` file
+in order for it to be opened without warnings. At Kitware, the cryptographic
+key used for this is located on a local physical device, where one person
+is in charge of performing all Windows signing.
```
Comment (cjh1) [inline docs/release/README.md:45]: We can remove the last sentence.

File: docs/release/notarization/README.md (hunk 1 above)
Comment (cjh1) [inline docs/release/notarization/README.md:13]: No sure, for now lets just remove this

File: docs/release/notarization/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+2. Create an Apple ID if you don't already have one
+3. Get added to an authorized provider (Kitware has a KitwareInc provider)
```
Comment (cjh1) [inline docs/release/notarization/README.md:15]: We can remove the bit about Kitware

File: docs/release/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+Create the Git Tag
+==================
+
+In the OpenChemistry/tomviz repository, create a signed git tag for the
+release/release candidate, and then push it. For instance:
```
Comment (cjh1) [inline docs/release/README.md:5]: Should be include updated the version in the CMake file?

**Comment by psavery** [comment]
@cjh1 I fixed the issues, and it is ready for review.

File: docs/release/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+git push --tags origin
+```
+
+Never delete a tag once pushed. If a mistake is made, make a new tag.
+
+
+Build the Binaries
+==================
+
+In in the OpenChemistry/tomviz-superbuild repository, trigger a manual "build"
```
Comment (cjh1) [inline docs/release/README.md:26]: typo

File: docs/release/notarization/post-notarization.tomviz [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+# fi
+
+# echo "Notarizing PKG"
+# xcrun altool --notarize-app --primary-bundle-id "org.tomviz.Tomviz" --username "${MY_APPLE_ID}" --password "${MY_APPLE_PASSWORD}" --asc-provider "KitwareInc" --file ${pkg_base}.pkg
+
+# echo "Notarization of .pkg requested.  Type 'done' when notarization email arrives."
+# while read line; do test "$line" == "done" && break; done
+
+# echo "Stapling .pkg"
+# xcrun stapler staple -v ${pkg_base}.pkg
```
Comment (cjh1) [inline docs/release/notarization/post-notarization.tomviz:51]: Can this just be removed?

Comment (psavery) [reply docs/release/notarization/post-notarization.tomviz:51]: I'm not sure. But I just removed it.

File: docs/release/README.md (hunk 5 above)
Comment (psavery) [reply docs/release/README.md:26]: Fixed.

**Review by cjh1 — APPROVED** [review]
LGTM

**Pull request merged**